src/test/java/org/ehcache/sizeof/SizeOfTestValues.java

```diff
@@ -41,6 +41,9 @@
             case 7:
                 hotspot32Bit.put("deepSizeOf(new ReentrantReadWriteLock())", 112L);
                 break;
+            case 8:
```

Comment:
I have no objection to using fallthrough here - makes it clearer what is going on imho.